### PR TITLE
fix(rpc/cli): separate inactive channel balance in getbalance

### PR DESCRIPTION
**Pull request by krrprr**
Closes https://github.com/ExchangeUnion/xud/issues/1130

### CODE REVIEW

File: lib/swaps/SwapClient.ts [hunk 1]
```diff
@@ -217,6 +219,8 @@ abstract class SwapClient extends EventEmitter {
     },
   ): Promise<void>;
 
+  public abstract listChannels(activeOnly: boolean): Promise<any>;
```
Comment (sangaman) [inline lib/swaps/SwapClient.ts]: I don't think we actually need this as a public generic method, at least for now. If you drop this you can also remove the stub implementation for the raiden class.

File: lib/raidenclient/RaidenClient.ts [hunk 2]
```diff
@@ -511,6 +511,10 @@ class RaidenClient extends SwapClient {
     return body.channel_address;
   }
 
+  public listChannels = (_activeOnly: boolean): Promise<Channel[]> => {
```
Comment (sangaman) [inline lib/raidenclient/RaidenClient.ts]: FWIW, Raiden does have a method to [list channels](https://raiden-network.readthedocs.io/en/stable/rest_api.html#querying-information-about-channels-and-tokens), however I'm not seeing how to determine whether the partner for a channel is online & active or not, so we couldn't calculate inactive balance as far as I can see anyhow.

File: test/jest/Service.spec.ts [hunk 3]
```diff
@@ -90,4 +93,94 @@ describe('Service', () => {
...
+      expect(btcBalance.inactiveChannelBalance).toEqual(18817);
+      expect(btcBalance.walletBalance).toEqual(10000);
+      expect(btcBalance.unconfirmedWalletBalance).toEqual(0);
+      expect(btcBalance.totalBalance).toEqual(289008);
+    });
+
+    test('throws in case of invalid currency', async () => {
+      setup();
+      try {
+        await service.getBalance({ currency: 'A' });
```
Comment (sangaman) [inline test/jest/Service.spec.ts]: As written I don't think this actually fails if `getBalance` doesn't throw. I believe you should use `await expect(...).rejects.toMatchSnapshot()` here and in the test case below.

Comment (krrprr) [reply test/jest/Service.spec.ts]: Indeed, good point!

**Pull request merged**